nodejs/awsx/ecs/ec2Service.ts

```diff
@@ -85,11 +85,18 @@ export class EC2Service extends ecs.Service {
             cluster.vpc, name, args.securityGroups || cluster.securityGroups, opts) || [];
         const subnets = args.subnets || cluster.vpc.publicSubnetIds;
 
+        const useClusterDefaultCapacityProviderStrategies = !!args.useClusterDefaultCapacityProviderStrategies;
```

Comment:
In https://github.com/pulumi/pulumi-awsx/issues/599#issuecomment-768688605 I wasn't really in favor of going down this approach as it feels like it's confusing to be able to specify `useClusterDefaultCapacityProviderStrategies` on _either_ `EC2Service` or `FargateService` with  effectively end up with the same behavior (not to mention the name `EC2Service` is now inappropriate because the underlying capacity provider may not actually be EC2). Any reason you didn't want to model it as a separate class as discussed in the comment?

Reply:
Yeap, I will comment in #599 to keep the conversation there. But I do agree with you.

Reply:
Now I see your point and it does make sense. I will create another PR with this. I will close this one since I no longer think this is the best option.